Spigot-Server-Patches/0419-Use-Object2BooleanOpenHashMap-and-ObjectSet-to-repla.patch

```diff
@@ -0,0 +1,81 @@
+From 161e41d8d883ba9d3bf1d86fa7138f578a25000c Mon Sep 17 00:00:00 2001
+From: Ghost_chu <2908803755@qq.com>
+Date: Fri, 4 Oct 2019 18:21:19 +0800
+Subject: [PATCH] Use Object2BooleanOpenHashMap and ObjectSet to replace the
+ HashMap and HashSet to optimize.
+
+Follow this thread https://www.spigotmc.org/threads/are-you-running-a-1-14-4-server-successfully.392911/#post-3534320, can see HashMap.remove took too many cpu times, so i have use FastUtil Object2BooleanOpenHashMap replace it to optimize the performance.
+
+diff --git a/src/main/java/net/minecraft/server/EntityTrackerEntry.java b/src/main/java/net/minecraft/server/EntityTrackerEntry.java
+index 460c687d..6aea39d1 100644
+--- a/src/main/java/net/minecraft/server/EntityTrackerEntry.java
++++ b/src/main/java/net/minecraft/server/EntityTrackerEntry.java
+@@ -7,6 +7,8 @@ import java.util.Iterator;
+ import java.util.List;
+ import java.util.Set;
+ import java.util.function.Consumer;
++
++import it.unimi.dsi.fastutil.objects.Object2BooleanOpenHashMap;
+ import org.apache.logging.log4j.LogManager;
+ import org.apache.logging.log4j.Logger;
+ 
+@@ -47,7 +49,7 @@ public class EntityTrackerEntry {
+         player.playerConnection.sendPacket(packet);
+     }
+ 
+-    public EntityTrackerEntry(WorldServer worldserver, Entity entity, int i, boolean flag, Consumer<Packet<?>> consumer, java.util.Map<EntityPlayer, Boolean> trackedPlayers) {
++    public EntityTrackerEntry(WorldServer worldserver, Entity entity, int i, boolean flag, Consumer<Packet<?>> consumer, Object2BooleanOpenHashMap<EntityPlayer> trackedPlayers) {
```

Comment:
This change is not necessary as Object2BooleanOpenHashMap is an instance of a map. Additionally this breaks ABI. Also missing paper comment